### PR TITLE
issue 4280: Remove the counts

### PR DESCRIPTION
**Pull request by zz9pzza**
https://code.google.com/p/otwarchive/issues/detail?id=4280

Remove the counts from the menu on every page.

### CODE REVIEW

File: app/views/user_sessions/_greeting.html.erb [hunk 1]
```diff
@@ -6,24 +6,12 @@
...
-          <% if @current_user.owned_collections.count > 0 %>
-            <li><%= span_if_current ts("My Collections (%{coll_number})", key: 'header', :coll_number => @current_user.owned_collections.count), user_collections_path(current_user) %></li>
-          <% end %>
-          <% if @current_user.challenge_signups.count > 0 %>
-            <li><%= span_if_current ts("My Sign-ups (%{signup_number})", key: 'header', :signup_number => @current_user.challenge_signups.count), user_signups_path(current_user) %></li>
-          <% end %>
-          <% if (@current_user.offer_assignments.undefaulted.count || @current_user.pinch_hit_assignments.undefaulted.count) > 0 %>
-            <li><%= span_if_current ts("My Assignments (%{assignment_number})", key: 'header', :assignment_number => (@current_user.offer_assignments.undefaulted.count + @current_user.pinch_hit_assignments.undefaulted.count)), user_assignments_path(current_user) %></li>
-          <% end %>
+          <li><%= span_if_current ts('My Subscriptions', key: 'header'), user_subscriptions_path(current_user) %></li>
```
Comment (sarken) [inline app/views/user_sessions/_greeting.html.erb]: This isn't quite what we agreed to -- we only agreed to stop displaying the count; we didn't even talk about displaying the non-relevant links. We'll still need `<% if @current_user.subscriptions.count > 0 %>` to make sure the user actually has Subscriptions and so on.

Comment (zz9pzza) [inline app/views/user_sessions/_greeting.html.erb]: If we are doing that then the counts have already been done and therefore on the ones that have conditionals we could put them back.

I would argue that we should have all the menu items there as it makes it obvious that the feature is available.

Comment (sarken) [inline app/views/user_sessions/_greeting.html.erb]: Can you please revive the discussion on the mailing list and explain what's going on here? None of us knew we would be agreeing to change how the menu works entirely, and I think our answers may change in light of this information. (I know personally, I would be okay with a compromise where we cache the counts for the conditional but didn't show the numbers in the link text. I wouldn't mind "My Works" lingering for a few minutes after I deleted my last work as much as I'd mind the number being wrong.)

**Comment by zz9pzza** [comment]
Will do so, I will close this up for the moment.

**Pull request closed without merging**